src/pages/home/report/ReportActionItemContentCreated.tsx

```diff
@@ -104,15 +104,17 @@ function ReportActionItemContentCreated({contextValue, parentReportAction, trans
         }
 
         return (
-            <ShowContextMenuContext.Provider value={contextValue}>
-                <View>
-                    <MoneyRequestView
-                        report={report}
-                        shouldShowAnimatedBackground
-                    />
-                    {renderThreadDivider}
-                </View>
-            </ShowContextMenuContext.Provider>
+            <OfflineWithFeedback pendingAction={action.pendingAction}>
```

Comment:
@dukenv0307 I think we should use `action.pendingAction` here because we use `action` for all `pendingAction` in this file like [here](https://github.com/Expensify/App/blob/b930641e35b207161b9be8885bafa2d0379f572d/src/pages/home/report/ReportActionItemContentCreated.tsx#L153)
https://github.com/Expensify/App/blob/b930641e35b207161b9be8885bafa2d0379f572d/src/pages/home/report/ReportActionItemContentCreated.tsx#L153